types/module/module.go

```diff
@@ -17,49 +17,54 @@ process. This separation is necessary, however we still want to allow for a
 high level pattern for modules to follow - for instance, such that we don't
 have to manually register all of the codecs for all the modules. This basic
 procedure as well as other basic patterns are handled through the use of
-ModuleBasicManager.
+BasicManager.
 */
-package types
+package module
 
 import (
 	"encoding/json"
 
-	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/gorilla/mux"
 	"github.com/spf13/cobra"
 	abci "github.com/tendermint/tendermint/abci/types"
-)
 
-// ModuleClient helps modules provide a standard interface for exporting client functionality
-type ModuleClient interface {
-	GetQueryCmd() *cobra.Command
-	GetTxCmd() *cobra.Command
-}
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
same as before.. `sdk` vs `types` or `sdktypes` for consistency?

Reply:
see previous comment